SoCCo/algorithms/climate.py

```diff
@@ -124,7 +125,86 @@ def perCapitaEmissionsToDelPPM(perCapCO2EmissionsN, popN,
     popTotal=7130010000 # Wolfram: QuantityMagnitude[CountryData["World", "Population"]]
     popN=popIntoNgroups(popTotal,nGroups=10)
     perCapitaEmissionsToDelPPM(pcE, popN)
+ 
+ #### pcEmissionsToIndex ########################################################
+
+def pcEmissionsToIndex(pcE, mean, sd):
+    """
+    Standardizes pcEmissions to a standard index using a normal cdf.
+    with a given mean and sd.
+    """  
     
+    pcE_Scaled=sp.norm(loc=mean,scale=sd).cdf(pcE)
+
+    return pcE_Scaled
+
+
+if False: # Using function
```

Comment:
This can be done in a different Pull request but these if False blocks will need to be removed. If you want to keep that code for reference, you could move it back to a script, which was your first implementation. That can be kept in its own file but this one is now a library module and should be kept as lean as possible.